client/src/components/IndividualReview.jsx

```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 
-const IndividualReview = () => (
-  <div className="inline-block">
-    <h6>IndividualReview</h6>
+const IndividualReview = ({review}) => {
+  return (
+    <div className="block">
+      <div className="inline-block">
+        <span>Star Rating: {review.star_rating}&nbsp;&nbsp;</span>
+        <span>Username: {review.review_username}&nbsp;&nbsp;</span>
+        <span>Date Posted: {review.review_date} months ago</span>
+        <p>Review title: {review.review_title}</p>
+        <span>
+          Review content: {review.review_content}
+        </span>
+        <p>Frequency of Use: {review.frequency_of_use}</p>
+        <p>Recommend this Product: {review.review_recommended}</p>
+        <h5>Helpful?</h5>
+        <span>Yes: {review.helpful_yes}&nbsp;&nbsp;</span>
+        <span>No: {review.helpful_no}&nbsp;&nbsp;</span>
+        <span>Report</span>
+      </div>
+      <div className="inline-block">
+        <p>QUALITY OF PRODUCT: {review.quality_rating}</p>
+        <p>VALUE OF PRODUCT: {review.value_rating}</p>
+      </div>
   </div>
-);
+  );
+};
```

Comment:
Above div and span structures are clean and easy to map how each tag would be rendered. Nice job.